tests/test_lhe_reader.py

```diff
@@ -218,3 +218,54 @@ def test_issue_102():
     assert len(list(pylhe.read_lhe(TEST_FILE_LHE_v3))) == len(
         list(pylhe.read_lhe_with_attributes(TEST_FILE_LHE_v3))
     )
+
+
+def test_read_lhe_init_raises():
+    """
+    Test that the <init> block is read correctly.
```

Comment:
Amend the doctring to state that the test raises an exception for missing block attributes.

It seems better to have the 2 tests separately, meaning the 2 raise. It's a minor thing, I know.